api/agentpb/agent.proto

```diff
@@ -283,6 +283,24 @@ message StartActionRequest {
     // Contains files and their contents which can be used in DSN.
     TextFiles text_files = 2;
   }
+  // PBMStatusParams describes pbm status action parameters.
+  message PBMStatusParams {
+    // DSN for the service. May contain connection (dial) timeout.
+    // May contain placeholders for file paths in DSN.
+    string dsn = 1;
+    // Contains files and their contents which can be used in DSN.
+    TextFiles text_files = 2;
```

Comment:
what is this? why we need some textfiles?

Reply:
That's for TLS certificates. Same field is present in all Mongo-related requests.